spec/src/main/asciidoc/ch11-metadata-for-or-mapping.adoc

```diff
@@ -1840,13 +1840,17 @@ and IDENTITY are not portable across all databases.]
 The further specification of table generators
 and sequence generators is described in <<a16164>> and <<a16256>>.
 
+The _UUID_ value indicates that the persistence provider should
+assign an RFC 4122 Universally Unique IDentifier.
+
 The _AUTO_ value indicates that the
 persistence provider should pick an appropriate strategy for the
 particular database. The _AUTO_ generation strategy may expect a
 database resource to exist, or it may attempt to create one. A vendor
 may provide documentation on how to create such resources in the event
 that it does not support schema generation or cannot create the schema
-resource at runtime.
+resource at runtime. In the case of a field or property of type
+_java.util.UUID_, the _AUTO_ strategy is equivalent to _UUID_.
```

Comment:
I'm wondering about the value of adding a UUID generator type enum value.  It would only be applicable to persistent fields of type UUID, and of little value to integer types.  If, for UUID types, AUTO == UUID, and with AUTO the default already for @GeneratedValue, then I would think that for the purpose of a generated value for a UUID type, just annotating it with @GeneratedValue would be sufficient.

Reply:
Nothing is conceptually incorrect about 

```
@Id
@GeneratedValue
@Uuid
String id;
```

Reply:
Or, along the lines of what the PR suggests

```
@Id
@GeneratedValue(UUID)
String id;
```